sphinx/templates/quickstart/conf.py_t

```diff
@@ -114,11 +114,8 @@ html_static_path = ['{{ dot }}static']
 # refs: http://alabaster.readthedocs.io/en/latest/installation.html#sidebars
 html_sidebars = {
     '**': [
-        'about.html',
```

Comment:
I think this is problematic because from [alabaster docs](http://alabaster.readthedocs.io/en/latest/installation.html#sidebars)

> the theme is designed assuming you’ll always have about.html activated, but otherwise it doesn’t care much.

So at least the `about.html` should be there commented-out, with some indication it is recommended for use with  alabaster.